lib/SILOptimizer/Mandatory/TFUtilities.cpp

```diff
@@ -323,7 +320,19 @@ static bool isZeroElementArrayAlloc(TupleExtractInst *tei) {
   if (!calleeName.contains("_allocateUninitialized"))
```

Comment:
contains or equals?

Reply:
contains.  This is a mangled name unfortunately.  I'm not sure what the best way is to model this.